services/virtual-assistant/src/virtual_assistant/watson.py

```diff
@@ -94,7 +101,7 @@ def create_session(self, session_id: str) -> str:
         return session_id
 
     def send_watson_message(
-        self, session_id: str, org_id: str, input: TalkInput
+        self, session_id: str, user_id: str, input: TalkInput
     ) -> dict:
         """Send a message to watson assistant
 
```

Comment:
I feel like the issue is not here. It's pobably setting to something different. I am investigating.

Reply:
Looks like we missed this one after a refactor, haha! We can keep this as is.